pkg/sql/distsqlrun/aggregator.go

```diff
@@ -147,7 +148,7 @@ func newAggregator(
 }
 
 // Run is part of the processor interface.
-func (ag *aggregator) Run(ctx context.Context, wg *sync.WaitGroup) {
+func (ag *aggregator) Run(ctx context.Context, wg *sync.WaitGroup, localStorage *engine.RocksDB, localStoragePrefix uint64) {
```

Comment:
What do you think of passing this in a processor's constructor rather than in Run(...). This would result in not having to change the processor interface and processors that will never use local storage.